src/modules/ie-objects/helpers/get-visitor-space-access-info-from-visits.ts

```diff
@@ -11,6 +11,6 @@ export const getVisitorSpaceAccessInfoFromVisits = (
 				?.filter((activeVisit: Visit) => activeVisit.accessType === VisitAccessType.Full)
 				?.map((activeVisit: Visit) => activeVisit.spaceMaintainerId) || [],
 		objectIds:
-			activeVisits?.flatMap((activeVisit: Visit) => activeVisit.collectionsIeSchemaIds) || [],
+			activeVisits?.flatMap((activeVisit: Visit) => activeVisit.accessibleFolderIds) || [],
```

Comment:
This puts folder ids into a variable named: objectIds
Folders !== Objects

So you probably have to also select the objects inside those folders